### PR TITLE
Updated CHANGELOG

### PR DESCRIPTION
**Pull request by nimzco**
Added v2.1.0 and v2.2.0 changelog

image

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Changelog
-## [Unreleased]
```
Comment (kaelig) [inline CHANGELOG.md]: It's typically a good idea to keep this line in, but commented out, to signify to the next developer they should put their changes under that section.

File: CHANGELOG.md [hunk 2]
```diff
@@ -34,4 +39,6 @@ import {autobind} from '@shopify/javascript-utilities/decorators';
 
 
 [Unreleased]: https://github.com/shopify/javascript-utilities/compare/v2.0.0...HEAD
```
Comment (kaelig) [inline CHANGELOG.md]: Update that diff link

File: CHANGELOG.md (hunk 1 above)
Comment (nimzco) [reply CHANGELOG.md]: Good idea, thanks 👍

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,14 @@
 # Changelog
-## [Unreleased]
+
+[//]: # (## [Unreleased])
```
Comment (kaelig) [inline CHANGELOG.md]: FYI this works too:

```markdown
<!-- ## [Unreleased] -->
```

Comment (nimzco) [reply CHANGELOG.md]: Good to know, the syntax I used is pretty weird 😬

**Pull request merged**